snowplow-tracker/src/main/java/com/snowplowanalytics/core/tracker/TrackerEvent.kt

```diff
@@ -69,4 +72,48 @@ class TrackerEvent @JvmOverloads constructor(event: Event, state: TrackerStateSn
         }
         return result
     }
+
+    fun addContextEntity(entity: SelfDescribingJson) {
+        entities.add(entity)
+    }
+
+    fun wrapContextsToPayload(payload: Payload, base64Encoded: Boolean) {
```

Comment:
nit: this could be `wrapEntitiesToPayload()`

Reply:
Good point, fixed!